TripPlanner - Project/APIsApp/test_routeapi.py

```diff
@@ -0,0 +1,65 @@
+    8.682461, 49.417389], [8.682563, 49.417388], [8.682676, 49.417387], [8.682782, 49.417388], [8.683371, 49.417368], [8.683592, 49.41736], [8.683763, 49.417362], [8.685222, 49.417366], [8.685359, 49.417364], [8.685511, 49.417366], [8.685713, 49.41737], [8.686395, 49.417364], [8.686722, 49.41737], [8.687335, 49.417352], [8.687474, 49.417348], [8.687598, 49.41735], [8.688256, 49.417361], [8.688813, 49.417381], [8.690001, 49.417413], [8.690227, 49.417428], [8.690338, 49.41743], [8.690482, 49.417401], [8.691467, 49.417155], [8.691735, 49.417102], [8.691957, 49.417117], [8.69198, 49.417121], [8.691941, 49.41722], [8.691817, 49.417369], [8.691427, 49.417726], [8.691072, 49.418051], [8.690968, 49.418158], [8.690936, 49.418188], [8.690939, 49.418208], [8.690939, 49.418296], [8.69092, 49.418378], [8.690912, 49.418411], [8.69067, 49.418981], [8.690664, 49.418992], [8.690614, 49.419093], [8.690547, 49.419174], [8.690479, 49.419204], [8.690446, 49.419218], [8.690275, 49.419577], [8.690123, 49.419833], [8.689854, 49.420216], [8.689652, 49.420514], [8.68963, 49.42051], [8.688601, 49.420393], [8.687872, 49.420318]], "type":"LineString"}}], "bbox": [8.681436, 49.41461, 8.69198, 49.420514], "metadata": {"attribution": "openrouteservice.org | OpenStreetMap contributors", "service": "routing", "timestamp": 1614100756243, "query": {"coordinates": [[8.681495, 49.41461], [8.687872, 49.420318]], "profile": "driving-car", "format": "json"}, "engine": {"version": "6.3.6", "build_date": "2021-02-21T01:31:06Z", "graph_date": "1970-01-01T00:00:00Z"}}}
+error_response = {"error": {"code": 2004, "message": "Request parameters exceed the server configuration limits. The approximated route distance must not be greater than 6000000.0 meters."}, "info": {
+    "engine": {"version": "6.3.6", "build_date": "2021-02-21T01:31:06Z"}, "timestamp": 1614100819473}}
+
+good_object = RouteApiRequest(8.681495, 49.41461, 8.687872, 49.420318)
+bad_object = RouteApiRequest(200, 200, 200, 200)
```

Comment:
It may be worth using pytest's fixtures to get and set up test data.

Reply:
Right, let's do it in a new ticket, @konecka-p would you like to do it?